sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileFormat.scala

```diff
@@ -171,6 +171,29 @@ trait FileFormat {
   def supportFieldName(name: String): Boolean = true
 }
 
+object FileFormat {
+
+  val FILE_PATH = "file_path"
+
+  val FILE_NAME = "file_name"
```

Comment:
wondering do we also plan to deprecate existing expression `InputFileName` in Spark?

Reply:
Good point. I think we should, as `InputFileName` is really fragile and can't be used with join for example.